chat-plugins/info.js

```diff
@@ -460,6 +288,13 @@ var commands = exports.commands = {
 		}
 		this.sendReply(buffer);
 	},
+	datahelp:
+		"/data [pokemon/item/move/ability] - Get details on this pokemon/item/move/ability/nature.\n" +
+		"!data [pokemon/item/move/ability] - Show everyone these details. Requires: + % @ & ~",
+	detailshelp:
+		"/details [pokemon] - Get additional details on this pokemon/item/move/ability/nature.\n" +
+		"!details [pokemon] - Show everyone these details. Requires: + % @ & ~",
+	dthelp: function () { this.parse('/help details'); },
```

Comment:
dt: 'details',
	details: function () {
		CommandParser.commands.data.call(this, arguments);
	},

This is the simple way to do it.